bindings/c/test/mako/mako.cpp

```diff
@@ -619,6 +619,19 @@ int workerProcessMain(Arguments const& args, int worker_id, shared_memory::Acces
 
 	selectApiVersion(args.api_version);
 
+	/* enable distributed tracing */
+	switch (args.distributed_tracer_client) {
+	case 1:
```

Comment:
Sorry, I'm late, but can we make these values into either macro or enum, so we can use human-friendly labels?

Reply:
Sure, will open a separate PR.